fe/fe-core/src/main/java/org/apache/doris/nereids/rules/expression/rules/JsonFunctionRewrite.java

```diff
@@ -106,4 +133,24 @@ private static <T extends ScalarFunction> Expression rewriteJsonModifyArguments(
         }
         return function.withChildren(convectedChildren);
     }
+
+    private static <T extends ScalarFunction> Expression rewriteJsonExtractFunctions(T function) {
+        JsonbExtract jsonExtract = new JsonbExtract(function.children().get(0), function.children().get(1));
```

Comment:
we better test the performance when rewriting json_extract_int/string to cast(json_extract ...), since before this PR, function just called once.

Reply:
and also test the performance after  get_json_xxx rewrite too